inspirehep/modules/forms/validators/simple_fields.py

```diff
@@ -157,44 +158,18 @@ def duplicated_arxiv_id_validator(form, field):
 
 
 def pdf_validator(form, field):
-    """Validate that url points to PDF."""
-    import requests
-
-    def get_content_type(url):
-        session = requests.Session()
-        try:
-            response = session.head(
-                url,
-                allow_redirects=True
-            )
-        except:
-            return
-        return response.headers['content-type']
-
-    message = "Please, provide an accessible direct link to a PDF document."
+    """Validate that the field contains a link to a PDF."""
+    message = 'Please, provide an accessible direct link to a PDF document.'
```

Comment:
I would go further and make this message for user firendly: "It looks like our server is not able to download this  document. Please provide an accessible direct link to a PDF document".